compose/config/config_schema_v2.0.json

```diff
@@ -191,6 +191,7 @@
         "restart": {"type": "string"},
         "security_opt": {"type": "array", "items": {"type": "string"}, "uniqueItems": true},
         "shm_size": {"type": ["number", "string"]},
+        "sysctls": {"$ref": "#/definitions/list_or_dict"},
```

Comment:
I think the support for the option has been added in API 1.24? In which case, this should be moved to the v2.1 config schema (for API 1.24+). v2.0 will need to stay compatible with API 1.22+